components/ContactListItem/index.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import Image from "next/image";
+import { StyledHR } from "../StyledHR";
+
+const StyledContainer = styled.div`
+  display: flex;
+  align-items: center;
+  /* justify-content: space-between; */
```

Comment:
You should remove this line, since it is out-commented, or?

Reply:
i do it